packages/cli/commands/project/listBuilds.js

```diff
@@ -0,0 +1,173 @@
+const path = require('path');
+
+const {
+  addAccountOptions,
+  addConfigOptions,
+  setLogLevel,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('@hubspot/cli-lib/errorHandlers');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  fetchProject,
+  fetchProjectBuilds,
+} = require('@hubspot/cli-lib/api/dfs');
+const {
+  getTableContents,
+  getTableHeader,
+} = require('@hubspot/cli-lib/lib/table');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { validateAccount } = require('../../lib/validation');
+const { link } = require('../../lib/links');
+const { getProjectConfig, getProjectDetailUrl } = require('../../lib/projects');
+const moment = require('moment');
+const { prompt } = require('inquirer');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list-builds [path]';
+exports.describe = false;
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: projectPath, limit } = options;
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('project-list-builds', { projectPath }, accountId);
+
+  const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();
```

Comment:
Maybe out of scope for this PR, but it looks like we could put this `const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();'  into the `getProjectConfig` util because we do it almost every time we call it.

Reply:
Handling this here https://github.com/HubSpot/hubspot-cli/pull/580/files#diff-fa4fa15b34bd2259fc2c24ec990fdbaa6492e6030d940700895318b399058174R64